client/src/components/Home/FindDogSitter.tsx

```diff
@@ -0,0 +1,47 @@
+import { useState } from 'react';
+import Box from '@material-ui/core/Box';
+import TextField from '@material-ui/core/TextField';
+import Button from '@material-ui/core/Button';
+import Stack from '@mui/material/Stack';
+import AdapterDateFns from '@mui/lab/AdapterDateFns';
+import LocalizationProvider from '@mui/lab/LocalizationProvider';
+import DatePicker from '@mui/lab/DatePicker';
+import { Typography } from '@material-ui/core';
```

Comment:
If you are importing the Typography as a named import, might as well import the above components within this import statement :)